### PR TITLE
Add verify debug output to connect command

### PR DESCRIPTION
**Pull request by csstaub**
r: @mcpherrinm @alokmenghrajani @christodenny 
Add verify debug output to connect command.

Examples:

Examples are from connecting to squareup.com, expired.badssl.com, wrong.host.badssl.com.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -49,9 +49,10 @@ var (
 	dumpFiles = dump.Arg("file", "Certificate file to dump (or stdin if not specified).").ExistingFiles()
 	dumpType  = dump.Flag("format", "Format of given input (heuristic guess if not specified).").String()
 
-	connect     = app.Command("connect", "Connect to a server and print its certificate.")
-	connectTo   = connect.Arg("server:port", "Hostname or IP to connect to.").String()
-	connectName = connect.Flag("name", "Override the server name used for SNI.").String()
+	connect       = app.Command("connect", "Connect to a server and print its certificate.")
+	connectTo     = connect.Arg("server:port", "Hostname or IP to connect to.").String()
+	connectName   = connect.Flag("name", "Override the server name used for SNI.").String()
+	connectCaPath = connect.Flag("ca", "Path to CA bundle (system default if unspecified).").ExistingFile()
```
Comment (alokmenghrajani) [inline main.go]: nit: display the system default path in this help output?

**Comment by alokmenghrajani** [comment]
👍

**Comment by mcpherrinm** [comment]
exploding-ship-gif

File: main.go (hunk 1 above)
Comment (csstaub) [inline main.go]: It's a system-specific value/function, built into the Go runtime. For example on macOS it's the keychain, whereas on Linux it's a distro-specific file path.

File: verify.go [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+		fmt.Fprintf(os.Stderr, "error reading CA bundle: %s\n", err)
+		os.Exit(1)
+	}
+
+	bundle := x509.NewCertPool()
+	bundle.AppendCertsFromPEM(bundleBytes)
+	return bundle
+}
+
+func verifyChain(certs []*x509.Certificate) {
```
Comment (mcpherrinm) [inline verify.go]: Can we pass in arguments here instead of using *connectName and *connectCaPath?

Comment (csstaub) [inline verify.go]: Can do

**Pull request merged**